ast/unify_test.go

```diff
@@ -31,7 +31,14 @@ func TestUnify(t *testing.T) {
 		{"object/var-3", `{"x": 1, "y": x} = y`, "[]", "[]"},
 		{"object/uneven", `{"x": x, "y": 1} = {"x": y}`, "[]", "[]"},
 		{"object/uneven", `{"x": x, "y": 1} = {"x": y}`, "[x]", "[]"},
-		{"call", "x = f(y)[z]", "[y]", "[x]"},
+		{"var/call-ref", "x = f(y)[z]", "[y]", "[x]"},
```

Comment:
nit: I could imagine adding negative cases where one or more of the input arguments are not safe and therefore the output should not include any new variables. I'd include cases w/ vars as args as well as arrays/sets/objects containing vars.